src/mobile/Studio/components/StudioInfo.tsx

```diff
@@ -0,0 +1,97 @@
+import { css } from '@emotion/react';
+import { colors, fonts } from '../../../styles/theme';
+
+interface studioInfoProps {
+  info: {
+    studio: string;
+    advisor: string;
+    description: string;
+    mobileImgSrc: string;
+  };
+}
+
+function StudioInfoMobile(studioInfo: studioInfoProps) {
+  const { studio, advisor, description, mobileImgSrc } = studioInfo.info;
+
+  return (
+    <section css={studioInfoContainer}>
+      <div css={mainCss}>
+        <div css={textCss}>
+          <h1 css={studioCss}>{studio}</h1>
+          <span css={advisorCss}>지도교수 {advisor}</span>
+        </div>
+        <img src={mobileImgSrc} css={imgCss} alt="스튜디오 대표 이미지" />
+      </div>
+      <p css={descriptionCss}>{description}</p>
+    </section>
+  );
+}
+
+export default StudioInfoMobile;
+
+const studioInfoContainer = css`
+  display: flex;
+  gap: 1.2rem;
+  flex-direction: column;
+
+  width: 100%;
+`;
+
+const mainCss = css`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const textCss = css`
+  display: flex;
+  gap: 0.8rem;
+  flex-direction: column;
+
+  padding-top: 1.6rem;
+`;
+
+const studioCss = css`
+  color: ${colors.gray900};
+
+  /* 모바일 */
+  @media (375px <= width < 768px) {
+    ${fonts.mobile_title_semi_20};
+  }
+
+  /* 태블릿 */
+  @media (768px <= width < 1440px) {
+    ${fonts.desktop_title_semi_24};
+  }
+`;
```

Comment:
도리가 말한 것처럼 이부분 아래처럼 수정해서 375 이하 너비도 화면을 띄워주기는 하도록 하는게 나을 것 같네용 !
```suggestion
  @media (width < 768px) {
    ${fonts.mobile_title_semi_20};
  }

  /* 태블릿 */
  @media (768px <= width < 1440px) {
    ${fonts.desktop_title_semi_24};
  }
`;
```